src/models/achievement.model.js

```diff
@@ -15,7 +15,8 @@ module.exports = function (app) {
     name: {type: String, required: true},
     current_amount: {type: Number, required: true},
     total_amount: {type: Number, required: true},
-    scope: {type: Map, required: false, default: null}
+    scope: {type: Map, required: false, default: null},
+    meta: {type: Map, required: false, default: {}}
```

Comment:
Saving `meta` to the database should not be strictly necessary. Instead, metadata could be fetched from the config everytime the achievement model is queried.